CHANGES/10055.misc.rst

```diff
@@ -0,0 +1,3 @@
+Downgraded logging of ``BadStatusLine`` exceptions to debug level -- by :user:`bdraco`.
+
+``BadStatusLine`` exceptions are relatively common, especially when connected to the public internet, because browsers or other clients may try to speak SSL to a plaintext server or vice-versa. These exceptions can quickly fill the log with noise when nothing is wrong.
```

Comment:
```suggestion
``BadStatusLine`` exceptions are relatively common, especially when connected to the public internet, because browsers or other clients may try to speak SSL to a plain-text server or vice-versa. These exceptions can quickly fill the log with noise when nothing is wrong.
```